### ISSUE TITLE
Needs a tutorial or blog on how one would use this.

### ISSUE THREAD

**Issue by rhatdan** [containers/bootc#10]
Would it be possible to build a base imae with osbuilder and push it to a container registry,
Add a layer to this image using buildah or podman build and update the image.

The use `bootc --install` to install the image on top of a VM or cloud image?

**Comment by cgwalters** [comment]
Yeah we need (and will hopefully soon have much more here) but the basic usage is touched on in current `README.md` - anywhere you have ostree (or `rpm-ostree`) it should work to just run the `bootc` binary (e.g. Fedora CoreOS, EndlessOS, a RHEL for Edge system, etc.)

I'll add a section on building custom base images.

**Comment by cgwalters** [comment]
:arrow_right: https://lists.centos.org/pipermail/centos-devel/2023-March/142809.html for CentOS

**Comment by cgwalters** [comment]
There's been a number of improvements here in git main; a ton more to do but I think we're in a better place.
